### PR TITLE
Mon 11220 unified sql

### PR DESCRIPTION
**Pull request by bouda1**
New module unified_sql to replace sql and storage modules.

REFS: MON-11220

### CODE REVIEW

File: bbdo/bam/dimension_kpi_event.cc [hunk 1]
```diff
@@ -16,7 +16,7 @@
 ** For more information : contact@centreon.com
 */
 
-#include "com/centreon/broker/bam/dimension_kpi_event.hh"
+#include "bbdo/bam/dimension_kpi_event.hh"
 
 #include "com/centreon/broker/database/table_max_size.hh"
```
Comment (Maaown) [inline bbdo/bam/dimension_kpi_event.cc:21]: wrong path

File: bbdo/storage/metric.cc [hunk 2]
```diff
@@ -16,25 +16,17 @@
 ** For more information : contact@centreon.com
 */
 
-#include "com/centreon/broker/storage/metric.hh"
+#include "bbdo/storage/metric.hh"
 
 #include <cassert>
 #include <cmath>
 
 #include "com/centreon/broker/database/table_max_size.hh"
```
Comment (Maaown) [inline bbdo/storage/metric.cc:24]: wrong path

File: bbdo/bam/dimension_kpi_event.cc (hunk 1 above)
Comment (bouda1) [reply bbdo/bam/dimension_kpi_event.cc:21]: image

Comment (bouda1) [reply bbdo/bam/dimension_kpi_event.cc:21]: Yes it exists

File: bbdo/storage/metric.cc (hunk 2 above)
Comment (bouda1) [reply bbdo/storage/metric.cc:24]: Yes!

File: bbdo/bam/dimension_kpi_event.cc (hunk 1 above)
Comment (bouda1) [reply bbdo/bam/dimension_kpi_event.cc:21]: OK, the generator is not called before the bbdo compilation.

File: centreon-engine/src/downtimes/service_downtime.cc [hunk 3]
```diff
@@ -18,9 +18,9 @@
  */
 
 #include "com/centreon/engine/downtimes/service_downtime.hh"
+#include <stdint.h>
```
Comment (Maaown) [inline centreon-engine/src/downtimes/service_downtime.cc:21]: maybe ``<cstdint>`` instead ?

**Review by Maaown — APPROVED** [review]

**Review by rem31 — APPROVED** [review]

**Pull request merged**